core/modules/qana/ColumnVertexMap.cc

```diff
@@ -160,4 +185,33 @@ std::vector<std::string> const ColumnVertexMap::computeCommonColumns(
     return cols;
 }
 
+
+std::ostream& operator<<(std::ostream& os, ColumnVertexMap::Entry const& e) {
+    os << "Entry(" << *e.cr << util::printable(e.vertices) << ")";
+    return os;
+}
+
+
+std::ostream& operator<<(std::ostream& os, ColumnVertexMap const& cvm) {
+    os << "ColumnVertexMap(" << util::printable(cvm._entries) << ")";
+    return os;
+}
+
+
+bool ColumnRefLt::operator()(query::ColumnRef const& a,
```

Comment:
these operator()s are moved from the header, and changed to use the column & table alias, instead of the db and table.